backend/src/main/java/gov/cdc/usds/simplereport/db/model/DeviceSpecimen.java

```diff
@@ -0,0 +1,41 @@
+package gov.cdc.usds.simplereport.db.model;
+
+import javax.persistence.Entity;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+
+import org.hibernate.annotations.NaturalId;
+
+/**
+ * A valid combination of device and specimen types. Can be soft-deleted, but
+ * cannot be otherwise modified.
+ */
+@Entity
+public class DeviceSpecimen extends EternalEntity {
```

Comment:
Technically if I merge first then _you_ will have fallen into the trap, but that would be kind of a dick move on my part. 😁 

I spent a significant chunk of last week wondering if I should have gone with DeviceSpecimenType instead of DeviceSpecimen, but resolved to just get the code written and wait and see what other people said. I would kind of have preferred it had nobody called me on that, so I could just walk away and not worry about it any more, but sadly I think you're right about the name. 😿